frontend/src/components/notFound/notFoundStyle.scss

```diff
@@ -0,0 +1,61 @@
+.not_found {
+  display: flex;
```

Comment:
You can use bootstrap for most of these custom styles. Try to use bootstrap classes instead of custom styles